googletest/README.md

```diff
@@ -80,13 +86,37 @@ using Visual Studio.
 
 On Mac OS X with Xcode installed, a `.xcodeproj` file will be generated.
 
+#### Incorporating Into An Existing CMake Project ####
+
+If you want to use gtest in a project which already uses CMake, then a
+more robust and flexible approach is to build gtest as part of that
+build directly. This has the significant advantage that the same
+compiler and linker settings are used between gtest and the rest of
+your project, so issues associated with using incompatible libraries,
+etc. are avoided. This is particularly useful on Windows.
+
+To achieve this, the project needs to know where to find the gtest
+sources. One approach is to embed the sources directly into your
+project's source tree, but this is not recommended. Using something
+like git submodules may be a suitable alternative for some projects,
+but a more flexible and less invasive approach is to get the CMake
+project to download/copy the gtest sources as part of the configure
+step, then pull that downloaded gtest directory into the build using
+`add_subdirectory()`. This technique is relatively straightforward
+and is fully explained in
```

Comment:
I'd like a quick example here so we're not completely dependent on the crascit article.